### PR TITLE
Remove secbuf and use nexus apis for token usage

### PR DESCRIPTION
**Pull request by modeveci**

### CODE REVIEW

**Comment by modeveci** [comment]
Related PRs and commits:

https://github.com/WebPlatformForEmbedded/Thunder/pull/260
https://github.com/WebPlatformForEmbedded/OCDM-Playready-Nexus-SVP/pull/11
https://github.com/Metrological/gstreamer-plugins-soc/commit/7139b6e80201a3fbc79a1c549f675c2d9349f80b
https://github.com/Metrological/gstreamer-plugins-soc/commit/4edcf29489e1dfa92c3a74fdcac4a7cf1b474288

File: MediaSession.cpp [hunk 1]
```diff
@@ -52,13 +52,25 @@ MediaKeySession::MediaKeySession(widevine::Cdm *cdm, int32_t licenseType)
...
+  NEXUS_Memory_Allocate(m_NexusMemorySize, nullptr, &m_pNexusMemory);
+  if (!m_pNexusMemory) {
+    printf("Memory allocation failure\n");
+    m_NexusMemorySize = 0;
+  }
 }
 
 MediaKeySession::~MediaKeySession(void) {
+
+    if (!m_pNexusMemory)
```
Comment (pawel-stanek-ml) [inline MediaSession.cpp]: you meant `if (m_pNexusMemory)` here

File: MediaSession.cpp [hunk 2]
```diff
@@ -303,6 +315,9 @@ CDMi_RESULT MediaKeySession::Decrypt(
     const uint8_t* keyId,
     bool /* initWithLast15 */)
 {
+
+  static NEXUS_HeapHandle secureHeap = NEXUS_Heap_Lookup(NEXUS_HeapLookupType_eCompressedRegion);
+
```
Comment (pawel-stanek-ml) [inline MediaSession.cpp]: Why is this static yet done for every Session::Decrypt() ?

File: MediaSession.cpp (hunk 1 above)
Comment (modeveci) [reply MediaSession.cpp]: ok, fixed

File: MediaSession.cpp (hunk 2 above)
Comment (modeveci) [reply MediaSession.cpp]: left as is

**Pull request merged**